clients/gitlabrepo/graphql.go

```diff
@@ -129,7 +129,7 @@ func (g *GitlabGID) UnmarshalJSON(data []byte) error {
 
 func (handler *graphqlHandler) getMergeRequestsDetail() (graphqlData, error) {
 	data := graphqlData{}
-	path := fmt.Sprintf("%s/%s", handler.repourl.owner, handler.repourl.project)
```

Comment:
I believe this one is still supposed to be `project` instead of `projectID`, this is causing a test failure here: https://github.com/ossf/scorecard/actions/runs/5017984263/jobs/8997813724?pr=3026#step:9:12047.

Reply:
Applied in latest commit.